src/components/MyAccount/NewSettings/NewAccountSettingsTab.tsx

```diff
@@ -0,0 +1,48 @@
+import { Flex, Banner } from "@nypl/design-system-react-components"
+import { useContext, useState } from "react"
+import { PatronDataContext } from "../../../context/PatronDataContext"
+import EmailForm from "./EmailForm"
+
+const NewAccountSettingsTab = () => {
+  const {
+    updatedAccountData: { patron },
+  } = useContext(PatronDataContext)
+  const [isSuccess, setIsSuccess] = useState(false)
+  const [isFailure, setIsFailure] = useState(false)
+
+  return (
+    <>
+      {isSuccess && (
+        <Banner
```

Comment:
The two `Banner`s are very similar expect for the text. Can you try to make the text and type conditional based on `isSuccess` and `isFailure` so that the component isn't repeated?